doc/md/cloud/agents.mdx

```diff
@@ -0,0 +1,204 @@
+to let RDS handle your database password in the Secrets Manager, you can provide the path to the actual token using a
+dot-notation. For example, for a secret like `{"password":"my_passw0rd!"}` you'd provide `.password` as the path to the token.
+
+</TabItem>
+<TabItem value="gcp_iam" label="GCP Cloud SQL Token (IAM Auth)">
+
+[GCP CloudSQL](https://cloud.google.com/sql) offers using
+[IAM Authentication](https://cloud.google.com/sql/docs/mysql/authentication#manual) to generate a short-lived token to
+use for authentication against a GCP CloudSQL database.
+
+1. Enable IAM Authentication for your database. For instructions on how to do this,
+   [see the GCP documentation](https://cloud.google.com/sql/docs/mysql/create-edit-iam-instances).
+
+2. Create a database user and grant it permission to authenticate using IAM, see
+   [the GCP documentation](https://cloud.google.com/sql/docs/mysql/add-manage-iam-users)
+   for instructions.
+
+All other info required to obtain a token is derived from the CloudSQL endpoint you define in the connection modal.
+
+</TabItem>
+<TabItem value="gcp_secrets" label="GCP Secrets Manager">
+
+GCP Secrets Manager is a secret store you can use to retrieve a secret from. Find the project ID and secret name of your
+secret. If you need more info, [read this guide](/guides/deploying/secrets##using-iam-authentication). If your secret is
+stored as a JSON, you can provide the path to the actual token using a dot-notation. For example, for a secret like
+`{"password":"my_passw0rd!"}` you'd provide `.password` as the path to the token.
+
+</TabItem>
+<TabItem value="env" label="Environment Variable">
+
+You can tell the agent to look for the password in an environment variable. For example, if your password lives in an environment
+variable like `DATABASE_PASSWORD=passw0rd` you'd provide `DATABASE_PASSWORD` to the agent.
+
+:::danger
+We advise to use either IAM authentication or a Secrets Manager to obtain a database password.
+:::
+
+</TabItem>
+</Tabs>
+
+To ensure the credentials are correct, Atlas will check if the credentials are working before we can save them. Hit
+the **Test Connection** button and wait. It can take a few seconds before the agent will pick up the job and check
+the connection to the database.
+
+If all goes well, we should see a message telling us that Atlas was able to connect to our database through the
+agent.
+
+<details>
+<summary>Screenshot Example</summary>
+
+![](https://atlasgo.io/uploads/cloud/drift-detection/6.png)
+</details>
+
+### Drift Detection
+
+Managing database schemas is hard. Since they are the (oftentimes only) stateful component in an application, issues
+originating in a faulty database state can have a wide impact on your products and services. A relatively simple issue
+that can have a huge impact is when the database schema is not in sync with what downstream consumers expect it to be.
```

Comment:
i would skip all of this and just define drift detection. also don't need to "simple" or "huge"